src/App.tsx

```diff
@@ -29,12 +29,19 @@ function App() {
   const { isMswReady, setIsMswReady } = useMswReadyStore((state) => state);
 
   useEffect(() => {
-    if (import.meta.env.VITE_APP_NODE_ENV === "development") {
-      setupMSW().then(() => setIsMswReady(true));
-    } else {
-      setIsMswReady(true);
+    async function initializeMSW() {
+      if (typeof window !== "undefined") {
+        if (import.meta.env.VITE_APP_NODE_ENV === "development") {
+          await setupMSW();
+          setIsMswReady(true);
+        } else {
+          setIsMswReady(true);
+        }
```

Comment:
위 if와 else 모두 setIsMswReady(true); 를 사용하는데 if문 밖에서 공통으로 처리하면 되지 않을까요? 아래 예시 코드 첨부합니다
```suggestion
        if (import.meta.env.VITE_APP_NODE_ENV === "development") {
          await setupMSW();
        }
        setIsMswReady(true);
```

Reply:
[c97dbf3](https://github.com/edu-pi/Front/pull/163/commits/c97dbf37e3a53cad8fed0eff0370548dc27c9a40)
그게 좋을 것 같네요 변경했습니다